kubectl-apply-all.sh

```diff
@@ -5,6 +5,10 @@
 # This file should be run:
 #   * When the cluster is first created
 #   * Whenever the configuration for any resource has been updated
-
+#
+# This --prune flag is destructive and should always be used 
```

Comment:
```suggestion
# The --prune flag is destructive and should always be used 
```

Reply:
also the label is crucial. it doesn't work well without a label. that's how it computes the diffs of what to keep and what to prune.